### DISCUSSION TITLE
Code button

### DISCUSSION THREAD

**Discussion by ctimmer** [micropython/micropython#9129] [General]
I don't know who controls the button behavior but when you press the code button '<>' it generates a single back quote for the beginning and ending of the code section. This should be 3 back quotes to start and 3 more to end the code section. The beginning back quotes can be followed immediately by the program language. 

Curt

**Comment by davefes** [comment]
Put all your code in, highlight it and then click the <>

**Reply by ctimmer** [reply]
Nice, thanks,
Curt

**Comment by davefes** [comment]
```
    w0 = network.WLAN(network.STA_IF)
#    w0.config(protocol=network.MODE_LR)
#  seems to be OK before
    w0.active(True)
```

**Reply by davefes** [reply]
OK, I only just figured that out yesterday :)

**Comment by jimmo** [comment]
Yes, just to clarify, the difference is between `print("inline code')` and:

```py
print("code block")
for i in range(10):
  print("hi')
```

Don't forget to add "py" after the `` ``` `` (i.e. `` ```py ``) to get Python syntax highlighting.

Thanks @davefes .

**Comment by davefes** [comment]
Even better, thanks
